### PR TITLE
crafted object data for export

### PR DESCRIPTION
**Pull request by Altair59**
added instructor, applicant, position, assignment, and ddah object data ready for export test

### CODE REVIEW

File: frontend/src/libs/exportUtils.ts [hunk 1]
```diff
@@ -41,6 +41,7 @@ function wageChunksMatchPositionAndSession(
     if (wageChunks.length === 1) {
         const chunk: WageChunk = wageChunks[0];
         if (
+            // shouldn't we also check chunk.hours === assignment.hours? Also sum of hours === assignment.hours when there are multiple chunks?
```
Comment (Altair59) [inline frontend/src/libs/exportUtils.ts]: @siefkenj Do we need to check chunk.hours === assignment.hours? Also sum of hours === assignment.hours when there are multiple chunks?

Comment (siefkenj) [reply frontend/src/libs/exportUtils.ts]: Yes, I think we probably should...

Comment (siefkenj) [reply frontend/src/libs/exportUtils.ts]: Remove this comment and make an issue for it.

Comment (Altair59) [reply frontend/src/libs/exportUtils.ts]: moved the issue to #504

**Comment by siefkenj** [comment]
Great job!

**Pull request merged**